src/routes/api/users.js

```diff
@@ -1,7 +1,7 @@
-const mongoose = require("mongoose");
+// const mongoose = require("mongoose");
 const router = require("express").Router();
 const passport = require("passport");
-const User = mongoose.model("User");
+// const User = mongoose.model("User");
```

Comment:
These comments don't serve any purpose and are therefore unnecessary, kindly remove them.